framework/base/Event.php

```diff
@@ -116,20 +116,20 @@ public static function off($class, $name, $handler = null)
         if ($handler === null) {
             unset(self::$_events[$name][$class]);
             return true;
-        } else {
-            $removed = false;
-            foreach (self::$_events[$name][$class] as $i => $event) {
-                if ($event[0] === $handler) {
-                    unset(self::$_events[$name][$class][$i]);
-                    $removed = true;
-                }
-            }
-            if ($removed) {
-                self::$_events[$name][$class] = array_values(self::$_events[$name][$class]);
-            }
+        }
 
-            return $removed;
+        $removed = false;
+        foreach (self::$_events[$name][$class] as $i => $event) {
+            if ($event[0] === $handler) {
+                unset(self::$_events[$name][$class][$i]);
+                $removed = true;
+            }
         }
+        if ($removed) {
+            self::$_events[$name][$class] = array_values(self::$_events[$name][$class]);
+        }
+
+        return $removed;
```

Comment:
rather not. Don't see the use of it as there is already an empty line because of the bracket